sample-apps/README.md

```diff
@@ -38,3 +38,6 @@ List of sample apps across all repositories in [aws-observability](https://githu
 |Sample App                   |Location                                                                                                                                    |App functionality                                                                                                                              |
 |-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
 |Ruby-manual-instrumentation  |[aws-otel-ruby](https://github.com/aws-observability/aws-otel-ruby/tree/main/sample-apps/manual-instrumentation/ruby-on-rails)              |Cotinuous integration of ADOT X-Ray components and X-Ray service. Manual Instrumentation using OpenTelemetry Ruby                              |
+
+Sample App Spec: 
```

Comment:
Was this supposed to be a link to the SampleAppSpec.md?

Reply:
Oh yes I forgot to add that